composer.json

```diff
@@ -305,6 +320,7 @@
             }
         },
         "enable-patching": true,
+        "composer-exit-on-patch-failure": false,
```

Comment:
@rajeshreeputra Let's revert this, otherwise our CI wouldn't exit and fail on patch failure.